### PR TITLE
UWP: Generate UWP Assets

### PR DESCRIPTION
**Pull request by alistair23**
Signed-off-by: Alistair Francis <alistair@alistair23.me>

### CODE REVIEW

File: src/UWP/Package.appxmanifest [hunk 1]
```diff
@@ -16,7 +16,8 @@
   <Applications>
     <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="Bit.UWP.App">
       <uap:VisualElements DisplayName="bitwarden" Square150x150Logo="Assets\Square150x150Logo.png" Square44x44Logo="Assets\Square44x44Logo.png" Description="bitwarden" BackgroundColor="transparent">
-        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png" ShortName="Bitwarden">
+        <uap:LockScreen Notification="badge" BadgeLogo="Assets\BadgeLogo.png" />
+        <uap:DefaultTile Wide310x150Logo="Assets\Wide310x150Logo.png" ShortName="Bitwarden" Square310x310Logo="Assets\LargeTile.png" Square71x71Logo="Assets\SmallTile.png">
```
Comment (kspearrin) [inline src/UWP/Package.appxmanifest]: `bitwarden` should be all lowercase.

**Comment by kspearrin** [comment]
Wow, that's a lot of assets. Did you use some tool to generate all these?

**Comment by alistair23** [comment]
It was all auto generated from Visual Studio. It generates all the UWP assets required for all sizes.

File: src/UWP/Package.appxmanifest (hunk 1 above)
Comment (alistair23) [reply src/UWP/Package.appxmanifest]: Ok, should be easy to find/replace

Comment (kspearrin) [reply src/UWP/Package.appxmanifest]: I only see this one line as needing an update.

**Comment by kspearrin** [comment]
I'll just fix the lowercase issue. I see it was like that before your change so no worries.

**Comment by alistair23** [comment]
Awesome. Thanks

**Pull request merged**